app/services/course/assessment/reminder_service.rb

```diff
@@ -32,9 +33,9 @@ def closing_reminder(assessment, token)
 
   def uncompleted_students(assessment)
     course_users = assessment.course.course_users
-    students = course_users.student.includes(:user).map(&:user)
+    students = course_users.student
     submitted =
-      assessment.submissions.confirmed.includes([course_user: :user]).map { |s| s.course_user.user }
+      assessment.submissions.confirmed.includes(:course_user).map(&:course_user)
```

Comment:
I think you still need to eager load the user ? cause it's used later in assessment_closing_reminder_email